db/seeds.rb

```diff
@@ -2,7 +2,8 @@
 require './db/seeds_files/companies'
 require './db/seeds_files/speakers'
 require './db/seeds_files/activities'
-require './db/seeds_files/badges16'
+require './db/seeds_files/badges17'
+require './db/seeds_files/users'
```

Comment:
Hm, can this use `require_relative 'seeds_files/<file>`?

Or even better: `Dir["db/seeds_files/*.rb"].each {|file| require file }`